v0/util/index.js

```diff
@@ -353,7 +365,12 @@ const handleMetadataForValue = (value, metadata) => {
         break;
       case "toNumber":
         formattedVal = Number(formattedVal);
-
+        break;
+      case "hashToSha256":
+        formattedVal = sha256(String(formattedVal));
+        break;
+      case "hashToSha256ForGender":
```

Comment:
not exactly a candidate for utils as it's very specific.
can we handle this with two transformations in metadata like ==> getFbGenderVal --> hashToSha256

Reply:
yes